app/paypal.py

```
@@ -0,0 +1,231 @@
+            access_token = response.json()['access_token']
+            if not access_token:
+                raise Exception('Invalid access token')
+            return access_token
+
+async def handle_transactions(params: tuple):
+    access_token = await login()
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"https://{PAYPAL_ENDPOINT}/v1/reporting/transactions", headers={
+            'Authorization': f"Bearer {access_token}",
+            'Content-Type': 'application/json',
+        }, params=params) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)